src/NHibernate/Hql/Ast/ANTLR/Tree/BinaryLogicOperatorNode.cs

```diff
@@ -200,17 +200,19 @@ public IParameterSpecification[] GetEmbeddedParameters()
 			return embeddedParameters.ToArray();
 		}
 
-		private string Translate(int valueElements, string comparisonText, string[] lhsElementTexts, string[] rhsElementTexts)
+		//TODO 6.0: Change to private protected (C#7.2 feature)
```

Comment:
You can change the project settings. To enable 7.2. I think the build agent should handle it.

Reply:
Done. Will see if all is OK with CI builds